cmd/wego/add/cmd.go

```diff
@@ -45,6 +45,7 @@ func init() {
 	Cmd.Flags().StringVar(&params.PrivateKey, "private-key", "", "Private key to access git repository over ssh")
 	Cmd.Flags().StringVar(&params.AppConfigUrl, "app-config-url", "", "URL of external repository (if any) which will hold automation manifests; NONE to store only in the cluster")
 	Cmd.Flags().BoolVar(&params.DryRun, "dry-run", false, "If set, 'wego add' will not make any changes to the system; it will just display the actions that would have been taken")
+	Cmd.Flags().BoolVar(&params.AutoMerge, "auto-merge", false, "If set, 'wego add' will merge automatically into the set --branch")
```

Comment:
minor thing, I see we used the `wego add will...` probably because of the dry run message, but I think that is redundant, you are already calling the `wego add` is implied that we are talking about it.

Reply:
It was because of dry run. I definitely copied it.